docs/user_guide/migrating_versions/index.rst

```diff
@@ -9,4 +9,6 @@ changes to the Fairlearn API.
 .. toctree::
    :maxdepth: 1
 
-   migrating_to_v0_5_0
\ No newline at end of file
+   migrating_to_v0_5_0
+   
+   migration_to_v0_3
```

Comment:
Should this be migrat*ing*_to_v0_3?

Reply:
Yes, it needs to match the filename

Reply:
Or maybe the `.. _migrating_to_v0_3_0:` header at the top of the corresponding file.... see my other comment.

Reply:
Hi! Thanks for the feedback I already changed it. I must've missed that I put migration instead of migrating.